examples/profile.php

```diff
@@ -70,89 +69,412 @@
                         <tr>
                             <th scope="row">Given Name(s)</th>
                             <td><?php echo $givenNames ? $givenNames->getValue() : '' ?></td>
                             <td>
-                                <?php echo ProfileHelper::getAttributeSources($givenNames) ?>
-                            </td>
-                            <td>
-                                <?php echo ProfileHelper::getAttributeVerifiers($givenNames) ?>
+                                <table class="table">
+                                    <thead class="thead-light">
+                                        <tr>
+                                            <th>S/V</th>
+                                            <th>Value</th>
+                                            <th>SubType</th>
+                                        </tr>
+                                    </thead>
+                                    <tbody>
+                                        <?php if ($givenNames): ?>
+                                            <?php foreach($givenNames->getSources() as $source): ?>
+                                                <tr>
+                                                    <td>Source</td>
+                                                    <td><?php echo $source->getValue() ?></td>
+                                                    <td><?php echo $source->getSubType() ?></td>
+                                                </tr>
+                                            <?php endforeach; ?>
+                                            <?php foreach($givenNames->getVerifiers() as $verifier): ?>
+                                                <tr>
+                                                    <td>Verifier</td>
+                                                    <td><?php echo $verifier->getValue() ?></td>
+                                                    <td><?php echo $verifier->getSubType() ?></td>
+                                                </tr>
+                                            <?php endforeach; ?>
+                                        <?php endif; ?>
+                                    </tbody>
+                                </table>
                             </td>
                         </tr>
                         <tr>
                             <th scope="row">Family Name</th>
                             <td><?php echo $familyName ? $familyName->getValue() : '' ?></td>
-                            <td><?php echo ProfileHelper::getAttributeSources($familyName) ?></td>
-                            <td><?php echo ProfileHelper::getAttributeVerifiers($familyName) ?></td>
+                            <td>
+                                <table class="table">
+                                    <thead class="thead-light">
+                                        <tr>
+                                            <th>S/V</th>
```

Comment:
Do you think it would be better to just have these column headers at the top, so we're not having to repeat them each time?

Reply:
Maybe, but And I remember Quirino said not have them as a column.